client/src/components/GameDetails/CreateGameForm.tsx

```diff
@@ -12,30 +13,38 @@ export const CreateGameForm:FC<CreateGameFormPropsT> = ({
   clickHandler,
 }) => {
   return (
-    <>
-      <input
-        type="number"
-        min={2}
-        max={5}
-        value={valueRoomLimit}
-        onChange={onChangeRoomLimit}
-        placeholder="Number of players"
-      />
-      <input
-        type="number"
-        min={10}
-        max={60}
-        value={valueGameDuration}
-        onChange={onChangeGameDuration}
-        placeholder="Game Duration"
-      />
-      <button
-        className={styles.buttonEnterName}
-        disabled={disabledBtn}
+    <div className={classes.inputWrapper} style={{flexDirection: 'column'}}>
+        <div>
+            <label>Number of players:</label>
+            <input
+                type="number"
+                min={2}
+                max={5}
+                value={valueRoomLimit}
+                onChange={onChangeRoomLimit}
+                placeholder="Number of players"
+                className={classes.inputNumber}
+            />
+        </div>
+        <div>
+            <label>Game Duration:</label>
+            <input
+                type="number"
+                min={10}
+                max={60}
+                value={valueGameDuration}
+                onChange={onChangeGameDuration}
+                placeholder="Game Duration"
```

Comment:
I think, **hardcoded strings** in `props` also need to move in the `constants` file.